moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -279,7 +261,7 @@ void ompl_interface::OMPLInterface::loadPlannerConfigurations()
       ROS_DEBUG_STREAM_NAMED(LOGNAME, " - " << parameters.first << " = " << parameters.second);
   }
 
-  setPlannerConfigurations(pconfig);
+  context_manager_.setPlannerConfigurations(pconfig);
```

Comment:
Why does the OMPL planner use a different approach to configure settings?
I would expect a similar approach to other planners, i.e. calling `setPlannerConfigurations(pconfig)`.

Reply:
I liked the way, ompl defined a default `setPlannerConfigurations` implementation to add empty configs for joint model groups and moved that implementation into the base class. But that did not work